src/components/screen/ProfileUpdate.tsx

```diff
@@ -103,6 +105,46 @@ function Screen(props: Props): React.ReactElement {
     }
   };
 
+  const uploadImage = async (uri: string): Promise<string> => {
+    interface FetchOption {
+      method: string;
+      body: {
+        inputFile: FormData;
+        dir?: string;
+      };
+      headers: Headers;
+    }
+
+    const fileName = uri.split('/').pop() || '';
+    const fileTypeMatch = /\.(\w+)$/.exec(fileName);
+    const data: FormData = new FormData();
+    const token = await AsyncStorage.getItem('token');
+    data.append('profile', {
+      uri: uri,
+      type: fileTypeMatch ? `image/${fileTypeMatch[1]}` : 'image',
+      name: fileName,
+    });
+
+    const fetchInitOption: FetchOption = {
```

Comment:
A variable doesn't need a type definition in most because of the dynamic typing of TypeScript.